pages/api/podcasts/transcript.tsx

```diff
@@ -12,13 +12,16 @@ import type {
   TranscriptDocument,
 } from 'types';
 import { createApiErrorResponse } from 'utils/createApiErrorResponse';
+import { tryConvertTextOrHtmlToVtt } from 'utils/tryConvertTextOrHtmlToVtt';
 
 const handler: NextApiHandler<TranscriptDocument | IApiErrorResponse> = async (
   req,
   res
 ) => {
   const url = typeof req.query.url === 'string' ? req.query.url : null;
-  const type = typeof req.query.type === 'string' ? req.query.type : null;
+  let type = typeof req.query.type === 'string' ? req.query.type : null;
+  const duration =
+    typeof req.query.duration === 'string' ? Number(req.query.duration) : null;
```

Comment:
I've been using `parseInt` instead of `Number()`. Probably best to stick to one. Any reason you chose `Number()`?

Reply:
I just remember reading somewhere to favor `Number()` these days...maybe something about perf and strictness. I'll switch to `parseInt()` since that's what the codebase does already.